README.md

```diff
@@ -193,38 +193,89 @@ const config: Config.InitialOptions = {
 export default config;
 ```
 
-### AskUIAnnotationStepReporter
+### AskUIXRayStepReporter
+
+> ❗️ **IMPORTANT NOTE**: Due to restrictions this reporter only works when you run your workflows in serial (default for AskUI)!
 
-#### Enable and Configure the AskUIAnnotationStepReporter in `jest.setup.ts`
+#### Enable and Configure the AskUIXRayStepReporter in `askui-helper.ts` (former `jest.setup.ts`)
+You have to do a few things in your `askui-helper.ts` (former `jest.setup.ts`) to enable the `AskUIXRayStepReporter`:
+
+1. Import the reporter
+2. Initialize the reporter outside the `beforeAll()` hook
+3. Add it to the `UiControlClient`
+4. Modify before `beforeEach()` and `afterEach()` hook to create/finish `TestEntries`
+5. Add writing the report to `afterAll()` hook
```

Comment:
Maybe not, although I am not a 100% certain, I think we can do most of this in the test environment (except getting the info of the steps themselves which we need to do in the `AskUIXRayStepReporter`). Doing this in the environment may also allow us to run tests parallely.

(This may be something for the next iteration instead of doing this a part of this PR here.)